WorkflowFunctionAdminBundle/Resources/views/AdministrationPanel/workflowFunctions.html.twig

```diff
@@ -3,9 +3,7 @@
     id="nav-workflow_functions"
     data-url="{{ path('open_orchestra_api_workflow_functions_list') }}"
     data-add="{{ path('open_orchestra_backoffice_workflow_function_new') }}"
```

Comment:
Same remark as on the master pr : can't the data-url and data-add be added to the conf to be removed here ?